### DISCUSSION TITLE
HELP Beta 13

### DISCUSSION THREAD

**Discussion by Francesco-Rugg** [apohl79/audiogridder#1024] [Help]
Hi, i had updated the stable version to beta 13 but the server is always on "Loading Plugin Cache" and the server not start.
When I'm click for "about" audio gridder starts but I can't connect to the server. My plugin finds my server but I cannot connect to it.
What I can do?
![Cattura](https://user-images.githubusercontent.com/62062196/189902405-48939747-7ce9-4698-833f-e62aadbedc88.JPG)

**Comment by apohl79** [comment]
You can reset AG's plugin cache by removing the files *.cache and *.layouts from the settings directory. See here: https://audiogridder.com/server/#config

**Reply by Francesco-Rugg** [reply]
Ok thank you, now I can connect to the server but I get this error
![Cattura](https://user-images.githubusercontent.com/62062196/189923472-593fe78f-f65c-44ca-a853-43d21f22b5b7.JPG)

**Reply by apohl79** [reply]
Will add some more diagnostics to the next build. I have never seen this happening.
